pkgs/development/cuda-modules/setup-hooks/auto-add-driver-runpath-hook.sh

```diff
@@ -0,0 +1,8 @@
+# shellcheck shell=bash
+# Run addDriverRunpath on all dynamically linked ELF files
+echo "Sourcing auto-add-driver-runpath-hook"
+
+if [ -z "${dontUseAutoAddDriverRunpath-}" ]; then
```

Comment:
From the other PR: https://github.com/NixOS/nixpkgs/blob/51557178a630c54cdab47edc0c366e54a29395db/pkgs/by-name/au/autoAddDriverRunpathHook/auto-add-driver-runpath-hook.sh#L25-L28